src/plone/volto/transforms.py

```diff
@@ -2,112 +2,29 @@
 from plone.restapi.deserializer.blocks import ResolveUIDDeserializerBase
 from plone.restapi.interfaces import IBlockFieldDeserializationTransformer
 from plone.restapi.interfaces import IBlockFieldSerializationTransformer
+from plone.restapi.interfaces import IBlockVisitor
 from plone.restapi.serializer.blocks import ResolveUIDSerializerBase
 from Products.CMFPlone.interfaces import IPloneSiteRoot
 from zope.component import adapter
-from zope.component import subscribers
 from zope.interface import implementer
+from zope.interface import Interface
 from zope.publisher.interfaces.browser import IBrowserRequest
 
 
-class NestedResolveUIDDeserializerBase(object):
```

Comment:
I should keep these base classes (but mark as deprecated) in case someone has registered custom adapters that extend them.